Assets/MixedRealityToolkit-SDK/Features/Input/MixedRealityInputManager.cs

```diff
@@ -1,4 +1,4 @@
-﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
oops, should this get reverted?

Reply:
I'm not even sure what changed. It should be okay.